src/style.css

```diff
@@ -0,0 +1,86 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 15px;
+  line-height: 20px;
+  text-decoration: none;
+}
+
+body {
+  padding-top: 250px;
+  background-color: #f4f5f7;
+}
+
+.container {
+  width: 50%;
+  margin: auto;
+  background-color: #fff;
+  border-radius: 3px;
+  box-sizing: border-box;
+  box-shadow: 0 2px 8px rgb(0 0 0 / 15%);
+}
+
+.header {
+  display: flex;
```

Comment:
You have done a great job,

Your code is clean and easy to navigate.

I noticed you used ` display: flex; padding: 1% 3% 1% 3%;` several times in your style.css, I think it would be to put all the classes that use the style together.

Something like this...

`.input,`
`.header,`
`.task {`
 `display: flex;`
 ` padding: 1% 3% 1% 3%;`
`}`